device/dhcp/servers/v6/dhcpv6server.yaml

```diff
@@ -0,0 +1,199 @@
+          $ref: '#/components/schemas/DhcpV6Server.Advanced'
+          x-field-uid: 5
+        leases:
+          description: |-
+            Array of DHCP pools configured on a server.            
+          type: array
+          items:
+            $ref: '#/components/schemas/DhcpV6Server.Lease'
+          x-field-uid: 6
+
+    DhcpV6Server.Lease:
+      description: >-
+        One DHCP pool configuration on a server.        
+      type: object 
+      required: [ia_type]
+      properties:
+        lease_time: 
+          description: >-
+            The Life Time length in seconds that is assigned to a lease if the requesting DHCP client does not specify a specific expiration time.
+          type: integer
+          format: uint32
+          minimum: 300
+          maximum: 30000000
+          default: 86400
+          x-field-uid: 1
+        ia_type:
+          $ref: '#/components/schemas/Dhcpv6Server.IaType'
+          x-field-uid: 5
+
+    Dhcpv6Server.IaType:
+      type: object
+      properties:
+        choice:
+          description: >-
+            Identity Association: a collection of leases assigned to a client. Each IA has an associated IAID. 
+            Each IA holds one type of lease, like an identity association for temporary addresses (IA_TA) holds 
+            temporary addresses, and an identity association for prefix delegation (IA_PD).
+          type: string
+          default: iana
+          x-field-uid: 1
+          x-enum:
+            iana:
+              x-field-uid: 1
+            iata:
+              x-field-uid: 2
+            iapd:
+              x-field-uid: 3
+            ianapd:
+              x-field-uid: 4
+        iana:
+          $ref: '#/components/schemas/Dhcpv6Server.PoolInfo'
+          x-field-uid: 2
+        iata:
+          $ref: '#/components/schemas/Dhcpv6Server.PoolInfo'
+          x-field-uid: 3  
+        iapd:
+          $ref: '#/components/schemas/Dhcpv6Server.PrefixPoolInfo'
+          x-field-uid: 4
+        ianapd:
+          $ref: '#/components/schemas/Dhcpv6Server.PoolInfo'
+          x-field-uid: 5
+
+    Dhcpv6Server.PoolInfo:
+      description: >-
+        The container for pool configurations for IA types iana,iata and ianapd.
+      type: object
+      properties:
+        start_address:
+          description: >-
+            The IP address of the first lease pool.
+          type: string
+          format: ipv6
+          default: ::a0a:101
+          x-field-uid: 1
+        prefix:
+          description: >-
+            The IPv6 network prefix length to be applied to the address. 
+          type: integer
+          format: uint32
+          default: 64
+          maximum: 128
+          x-field-uid: 2
+        size:
+          description: >-
+            The total number of addresses in the pool.
+          type: integer
+          format: uint32
+          default: 10
+          minimum: 1
+          maximum: 1000000
```

Comment:
This maximum is missing for IPv4 DHCP . Probably should be there ?

Reply:
will add

Reply:
added in v4